hazelcast/src/main/java/com/hazelcast/map/impl/operation/ClearOperation.java

```diff
@@ -70,9 +70,11 @@ public void applyState(State state) {
 
     @Override
     public void afterRunInternal() {
+        if (recordStore == null) {
+            return;
+        }
```

Comment:
Oops, seems i misled you.
This new snippet below seems the correct change for both operations by preserving old behavior.

```suggestion
        @Override
    public void afterRunInternal() {
        if (mapContainer != null) {
            invalidateAllKeysInNearCaches();
        }
        hintMapEvent();
    }
```

Reply:
https://github.com/hazelcast/hazelcast/commit/0cf38e8ea64d46c7d044da42939c1e05545de303